blob-tool/src/main.rs

```diff
@@ -63,7 +63,9 @@ async fn main() {
     );
 
     let index = blob.index.unwrap();
-    let first_row_index: u64 = index.div_ceil(eds_size) - 1;
+    //let first_row_index: u64 = index.div_ceil(eds_size) - 1;
+    // Trying this Diego's way
+    let first_row_index: u64 = blob.index.unwrap() / eds_size;
```

Comment:
from https://github.com/eigerco/lumina/blob/e1a7644a393ac018642e7faadd64d5fad581f3b7/types/src/blob.rs#L44-L47 


```rust
    /// Index of the blob's first share in the EDS. Only set for blobs retrieved from chain.
    // note: celestia supports deserializing blobs without index, so we should too
    #[serde(default, with = "index_serde")]
    pub index: Option<u64>,
```

I am not familiar with if this is even `None`, but I would guess it can be. Should we handle the error instead here? or map to a default?

Reply:
Eliminating all paths to panic is the right approach, so we can `map_err` to something just for consistency, even though this should never be None.